libs/accounts/recovery-phone/src/lib/recovery-phone.service.spec.ts

```diff
@@ -421,7 +421,7 @@ describe('RecoveryPhoneService', () => {
       expect(service.hasConfirmed(uid)).rejects.toEqual(
         new RecoveryPhoneNotEnabled()
       );
-      expect(() => service.maskPhoneNumber('+15550005555')).toThrow(
+      expect(() => service.stripPhoneNumber('+15550005555')).toThrow(
```

Comment:
In someone ways I'd prefer both things to exist. Seems like UX flip flops a lot, and additive changes are easier to track.

Reply:
I'm not sure if I like the back-end handling adding the bullet points. Since that feels more presentation layer to me, I think it's nicer for the front-end to handle it - so if we want to change this in the future, we can in those new `recovery-phone-utils` functions.